### PR TITLE
Adding admin users for repository request

### LINKED ISSUE #66: Adding admin users for repository request
I want to add one or multiple GitHub users to add them as admins during the creation of the repository.

### PR DESCRIPTION
**Pull request by spindev**
closes #66

### CODE REVIEW

**Comment by m4s-b3n** [comment]
the original intention was to use the [metadata](https://github.com/xpirit-training/demo-features-issueops/blob/main/org/companies/Xpirit/teams/Team%201/metadata.yml#L2) for this purpose. manager could be turned into an array here (and remove the @)

**Comment by m4s-b3n** [comment]
would start with that...ok 4 u?

**Comment by aatmmr** [comment]
@m4s-b3n how about both? The metadata holds the default admins for team repos. Additional admins could be added via issue form (respective about default admins should be added to issue form).

Are we supporting teams as best practice?

**Comment by spindev** [comment]
> @m4s-b3n how about both? The metadata holds the default admins for team repos. Additional admins could be added via issue form (respective about default admins should be added to issue form).
> 
> Are we supporting teams as best practice?

We should, but not for now 😢

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
 	"features": {
 		"ghcr.io/devcontainers-contrib/features/actionlint:1": {},
 		"ghcr.io/guiyomh/features/vim:0": {},
-		"ghcr.io/eitsupi/devcontainer-features/jq-likes:1": {},
+		"ghcr.io/eitsupi/devcontainer-features/jq-likes:1": {
+			"jqVersion": "os-provided",
```
Comment (m4s-b3n) [inline .devcontainer/devcontainer.json:11]:
```suggestion
			"jqVersion": "latest",
```

File: .github/workflows/repo-creation.yml [hunk 2]
```diff
@@ -115,6 +125,24 @@ jobs:
...
+      - name: Get project admins
+        run: |
+          
+      - name: Add admins to repo
+        id: add-admins
+        if: env.ADMINS
+        env:
+          GH_TOKEN: ${{ steps.get-workflow-token.outputs.token }}
+        run: |
+          for admin in $(echo $ADMINS | tr ", " "\n")
```
Comment (m4s-b3n) [inline .github/workflows/repo-creation.yml:137]: what if they enter the list like `user1,user2,user3` or `user1,  user2,  user3`?

this is why i would not use text fields for this but yaml only

File: org/companies/MariBo GmbH/teams/MeOnly/metadata.yml
Comment (m4s-b3n) [inline org/companies/MariBo GmbH/teams/MeOnly/metadata.yml:1]: some teams are missing

**Pull request closed without merging**